src/items/constant-items.md

```diff
@@ -48,9 +48,6 @@ const BITS_N_STRINGS: BitsNStrings<'static> = BitsNStrings {
 };
 ```
 
-r[items.const.final-value-immutable]
-The final value of a `const` item cannot contain references to anything mutable.
-
```

Comment:
We were wondering if it would make sense to change the wording here instead of removing it. Perhaps something like:

"The final value of a `const` item cannot contain a mutable reference."

Most of us seemed to think this is correct. Perhaps there is some other rule forbidding this already (implicitly or explicitly), but it seems like it wouldn't hurt to be clear about it here?